.buildkite/scripts/tooling.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+retry() {
```

Comment:
Currently, this function looks like is not being used.
Do you think it can be removed along with the `source` commands of this file?